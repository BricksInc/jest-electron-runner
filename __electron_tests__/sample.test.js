/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */

test('test', async () => {
  expect(1).toBe(1);
  expect(window).toBeDefined();
  window.abc = 11111111111111111;
  expect(window.abc).toMatchSnapshot();
  return new Promise(r => {
    setTimeout(r, 500);
  });
});
