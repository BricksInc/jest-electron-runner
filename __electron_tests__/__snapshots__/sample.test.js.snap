// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `11111111111111112`;
